src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -1,21 +1,42 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags file_tags grid_tags table_tags solo_tags form_tags button_tags %}
+{% load i18n grid_tags table_tags button_tags icon_tags %}
 
 {% block content %}
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                {# Contactmoment/dashboard. #}
-                <h1 class="h1" id="title">{{ page_title }}</h1>
+                <h1 class="h1" id="title">{{ contactmoment.onderwerp }}</h1>
                 {% include "components/Dashboard/Dashboard.html" with metrics=metrics only %}
             {% endrender_column %}
-            {% render_column start=4 span=6 %}
+        {% endrender_grid %}
 
-                <div class="contactmoment__details">
-                {% contactmoment_table contactmoment zaak=zaak %}
+        <div class="container--no-margin">
+            <div class="contactmoment">
+                <h3>{% trans "Uw vraag" %}</h3>
+                <p>{{ contactmoment.text }}</p>
+            </div>
+            <div class="contactmoment">
+                <h3>{% trans "Antwoord" %}</h3>
```

Comment:
The datum/date is missing here - you can use a H2 element that's wrapping an anchor just like we are using on the Homepage.
Look at the design: 
https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=1625-11082&mode=design&t=irQankQdfNj4iA6o-0

Reply:
We don't have the data available. This is the second place where we're departing from the design.